### PR TITLE
Corrected Lucky Day (ID: 4880) item script

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: #4273 

* **Server Mode**: renewwal

* **Description of Pull Request**: 
The effect now procs when being attacked and when dealing damage
The rates are changed from 1% to 1.5%

Thanks to @Questune09 !

### CODE REVIEW

**Comment by attackjom** [comment]
https://www.youtube.com/watch?v=pvGLvmVuKEA&t=214s

From this video in 7.57 

When activation lucky day, it show the MVP Banner specialeffect.

File: db/re/item_db.txt [hunk 1]
```diff
@@ -3059,7 +3059,7 @@
 4877,Speed_Of_Light,Speed of Light,6,20,,10,,,,,,,,,,,,,{ /*TODO: Confirm the rate*/ autobonus "{ bonus bAspdRate,100; bonus bFlee,100; bonus2 bHPLossRate,400,1000; bonus2 bSPLossRate,50,1000;}",10,5000,BF_WEAPON,"{ specialeffect2 EF_AGIUP2; }"; },{},{ heal 0,-300; }
 4878,Muscle_Fool,Muscle Fool,6,20,,10,,,,,,,,,,,,,{ autobonus2 "{ bonus bVit,200; bonus2 bAddRace,RC_All,-50; bonus bMatkRate,-50; }",10,5000,BF_WEAPON,"{ specialeffect2 EF_GUARD3; }"; },{},{ heal 0,-300; }
 4879,Hawkeye,Hawkeye,6,20,,10,,,,,,,,,,,,,{ autobonus "{ bonus bDex,200; bonus2 bSPLossRate,50,1000; }",10,5000,BF_WEAPON,"{ specialeffect2 EF_BASH3D6; }"; },{},{ heal 0,-300; }
-4880,Lucky_Day,Lucky Day,6,20,,10,,,,,,,,,,,,,{ autobonus "{ bonus bLuk,200; }",10,5000,BF_WEAPON|BF_MAGIC,"{ specialeffect2 EF_DANCE_BLADE_ATK; }"; },{},{ heal 0,-300; }
+4880,Lucky_Day,Lucky Day,6,20,,10,,,,,,,,,,,,,{ autobonus "{ bonus bLuk,200; }",15,5000,BF_WEAPON|BF_MAGIC,"{ specialeffect2 EF_MVP; }"; autobonus2 "{ bonus bLuk,200; }",15,5000,BF_WEAPON|BF_MAGIC,"{ specialeffect2 EF_MVP; }"; },{},{ heal 0,-300; }
```
Comment (cydh) [inline db/re/item_db.txt]: wait, the attacked rate is 0.1% (checked by script, and by experience I never had chance to trigger LD whle being attacked)
and it's only physical for both.

**Comment by kyurejl** [comment]
autobonus "{ bonus bLuk,200; bonus2 bAddMonsterDropItem,7444,100; }",15,5000,BF_WEAPON|BF_MAGIC,"{ specialeffect2 EF_DANCE_BLADE_ATK; }"; autobonus2 "{ bonus bLuk,200; bonus2 bAddMonsterDropItem,7444,100; }",15,5000,BF_WEAPON|BF_MAGIC,"{ specialeffect2 EF_DANCE_BLADE_ATK; }";

credits to: https://www.shining-moon.com/?module=preitem&action=view&id=4880

**Comment by kyurejl** [comment]
autobonus "{ bonus bLuk,200; bonus2 bAddMonsterDropItem,7444,100; }",15,5000,BF_WEAPON|BF_MAGIC,"{ specialeffect2 EF_DANCE_BLADE_ATK; }"; autobonus2 "{ bonus bLuk,200; bonus2 bAddMonsterDropItem,7444,100; }",15,5000,BF_WEAPON|BF_MAGIC,"{ specialeffect2 EF_DANCE_BLADE_ATK; }";

credits to: https://www.shining-moon.com/?module=preitem&action=view&id=4880

**Pull request merged**